MapboxVision/VisionManager/VisionManager.swift

```diff
@@ -266,18 +299,19 @@ extension VisionManager: VideoSourceObserver {
 extension VisionManager: RecordCoordinatorDelegate {
     func recordingStarted(path: String) {}
 
-    func recordingStopped() {
+    func recordingStopped(recordingPath: RecordingPath) {
+        try? handle(recordingPath: recordingPath, country: currentCountry)
```

Comment:
Here we should handle not current country, but the country which was known before the moment of stopping, we can assume that if before stopping there was country `.unknown` it is equal to the new one